src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethodPublicOnly1.java

```diff
@@ -133,7 +133,7 @@ public void run() {
             /**
              * Javadoc
              */
-            private String privateMethod(String a, String b) {
+            private String privateMethod(String ax, String bx) {
```

Comment:
Please restore this code.
We do not change Inputs until absolutely required. All Input files are very weird and bad and this is exactly what we want

Reply:
Done @romani